rpcs3/Emu/Cell/SPUThread.cpp

```diff
@@ -3850,6 +3850,7 @@ s64 spu_thread::get_ch_value(u32 ch)
 
 		const s64 out = channel.pop_wait(*this);
 		static_cast<void>(test_stopped());
+		wakeup_delay();
```

Comment:
Added it here for Signal Notify Registers because they are used unofficially in games for SPU tasks as well.